packages/plugin/src/classes/helpers/classes.js

```diff
@@ -222,39 +222,24 @@ export function convertClassToUI5Extend(
       // @transformControllerExtension marker, because it is not a type assignment. In the resulting code, the
       // "ControllerExtension.use(...)" part should be removed and the content of the brackets should be assigned
       // directly to the member property.
-      if (t.isCallExpression(member.value)) {
-        const callee = member.value.callee;
-        if (
-          t.isMemberExpression(callee) &&
-          t.isIdentifier(callee.object) &&
-          t.isIdentifier(callee.property) &&
-          callee.property.name === "use" // we are looking for "ControllerExtension.use(...)"
-        ) {
-          const importDeclaration = getImportDeclaration(
-            memberPath?.hub?.file?.opts?.filename,
-            callee?.object?.name // usually, but not necessarily always: "ControllerExtension"...
-          );
-          // ...hence we rather look at the imported module name to be sure
-          if (
-            importDeclaration?.source?.value ===
-            "sap/ui/core/mvc/ControllerExtension"
-          ) {
-            if (
-              !member.value.arguments ||
-              member.value.arguments.length !== 1
-            ) {
-              // exactly one argument must be there
-              throw memberPath.buildCodeFrameError(
-                `ControllerExtension.use() must be called with exactly one argument but has ${
-                  member.value.arguments ? member.value.arguments.length : 0
-                }`
-              );
-            }
-            member.value = member.value.arguments[0];
-            extendProps.unshift(buildObjectProperty(member)); // add it to the properties of the extend() config object
-            continue; // prevent the member from also being added to the constructor
-          }
-        }
+      const rightSide = member.value;
+      if (isCallToControllerExtensionUse(rightSide, memberPath)) {
+        member.value = rightSide.arguments[0];
+        extendProps.unshift(buildObjectProperty(member)); // add it to the properties of the extend() config object
+        continue; // prevent the member from also being added to the constructor
+      }
+
+      // code instrumentation sometimes wraps ControllerExtension.use() calls like:
+      // this.routing = (cov_1uvvg22e7l().s[5]++, ControllerExtension.use(Routing.override({ … })));
+      if (
+        t.isSequenceExpression(rightSide) &&
+        rightSide.expressions.some((expression) =>
```

Comment:
Only the last entry in the sequence is used - why do you use the `some` to find the controller extension instead of just checking the last entry?

Reply:
Because ALL ControllerExtension.use() calls need to be replaced (would otherwise cause a runtime error) - of course it's hard to see how they would get there without intention.
But actually this removal was not done in a SequenceExpression - the "expected" test result was wrong. Fix coming.

Reply:
Or we just ignore this case... Even though also the call in earlier places in the sequence will be executed, only in the last place the result will be actually used, so we an expect that it only occurs in the last place. This makes the code simpler. We also don't replace it in any other places in the code, anyway. Then also not in those earlier slots of the sequence.